### ISSUE TITLE
Headings are not structured (Accessibility)

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/SearchWorks#3782]
One or more heading levels have been skipped.

[SiteImprove Issue with details](https://my2.siteimprove.com/Accessibility/1059585/NextGen/Issue/1?ruleId=53&pageSegments=&issueKind=1&tagFilterType=2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2)

This issue is referencing the record view - specifically, records with both an online and a physical item option. "Available online" is an H3 and it appears after the record title, which is an H1, therefore skipping a level. See screenshots for details.

Sidenote: I feel "At the library" (H2) gets lost because the H3 headings have much more weight. It might be helpful to play around with the styling a bit on the record view to enhance the readability/scanability using visual hierarchy.

### Example of record with issue - both an online and physical item option

![Image](https://github.com/sul-dlss/SearchWorks/assets/6816624/7d8ecb0d-e783-488d-a615-585df55fd767)

### Example of record without issue - only a physical item option

![Image](https://github.com/sul-dlss/SearchWorks/assets/6816624/600bb1a3-f368-4d2e-b8fa-db32f33bfcde)

**Comment by jcoyne** [comment]
@dbranchini Would making "Available online" an `<aside>` or a `<nav>` with a `<header>` resolve this?

**Comment by dnoneill** [comment]
@dbranchini Would it make sense to make this view match all the other views for online items? See attached screenshot.



![Image](https://github.com/sul-dlss/SearchWorks/assets/19173991/5a126f41-f513-47d2-888b-6d848f24338d)

**Comment by jcoyne** [comment]
@dnoneill I don't think that proposal works as "More options" is not "Online".  For example. Interlibrary loan shows up in the "More options"

**Comment by dnoneill** [comment]
@jcoyne I am not sure I understand. The screenshot I attached is of an [existing record](https://searchworks.stanford.edu/view/14434594) and how it is showing up now. I definitely could see how there could be something I am missing because of this line which I am not sure why it was written to exclude the header if there were links: https://github.com/sul-dlss/SearchWorks/blob/b669d3ec5ee48d87c6fb56a7dd5948ad3bf57a6e/app/components/access_panels/online_component.html.erb#L4

**Comment by dbranchini** [comment]
In my opinion, the H2, "At the Library" looks disjointed from the "Other options" under it. The weighting of the H2 and H3 elements seems off to me, creating a lack of visual hierarchy.  For a short-term fix, <aside> should work. I haven't tried it yet though.

**Comment by dbranchini** [comment]
I'm talking to Sarah about possibly removing some of the H2 headings (mainly At the library, Context) from the sidebar as a solution instead of adding more. I'm going to do a couple of quick mocks to share with Chris and others before we decide.

**Comment by jcoyne** [comment]
@dbranchini what are you referring to when you say "let's do that?"

**Comment by dbranchini** [comment]
I meant Niqui's suggestion above to add the H2 Online heading as shown in the screenshot above and her screenshot.
